docs/basics/installation.rst

```diff
@@ -46,41 +91,205 @@ You can find detailed installation instructions on how to do this
+   `WSL troubleshooting page <https://docs.microsoft.com/en-us/windows/wsl/troubleshooting>`_.
+
+
+   - **Step 1**: Enable the windows subsystem for Linux
+
+      - Open Windows Power Shell as an Administrator and run
+
+      .. code-block:: bash
+
+         $ Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+
+      - Afterwards, when prompted in the Power Shell, restart your computer
+
+   - **Step 2**: Install a Debian Linux distribution
+
+      - To do this, visit the Microsoft store, and search for the Debian distro.
+        We **strongly** recommend installing :term:`Debian`, even though other
+        distributions are available. "Get" the app, and "install" it.
+
+   - **Step 3**: Initialize the distribution
+
+      - Launch the Subsystem either from the Microsoft store or from the Start menu. This
+        will start a terminal. Don't worry -- there is a dedicated section (:ref:`howto`)
+        on how to work with the terminal if you haven't so far.
+
+      - Upon first start, you will be prompted to enter a new UNIX username and password.
+        Tip: chose a short name, and no spaces or special characters. The password will
+        become necessary when you elevate a process using ``sudo`` -- sudo let's you execute a
+        process with rights of another user, such as administrative rights, for examples when
+        you need to install software.
+
+      - Right after initial installation, your Linux distribution will be minimally equipped.
+        Update your package catalog and upgrade your installed packages by running the command below.
+        As with all code examples in this book, make sure to copy commands exactly, including
+        capitalization. If this is the first time you use ``sudo``, your system will warn you
+        to use it with care. During upgrading installed packages, the terminal will ask
+        you to confirm upgrades by pressing ``Enter``.
+
+      .. code-block:: bash
+
+         $ sudo apt update && sudo apt upgrade
+
+   - **Step 4**: Enable Neurodebian
+
+      - In your terminal, run
+
+      .. code-block:: bash
+
+         $ wget -O- http://neuro.debian.net/lists/stretch.de-md.libre | sudo tee /etc/apt/sources.list.d/neurodebian.sources.list
+
+      - Afterwards, run
+
+      .. code-block:: bash
+
+         $ curl -sL "http://keyserver.ubuntu.com/pks/lookup?op=get&search=0xA5D32F012649A5A9" | sudo apt-key add
+
+      - lastly do another
+
+      .. code-block:: bash
+
+         $ sudo apt-update && sudo apt upgrade
+
+   - **Step 4**: Install datalad and everything it needs
+
+      .. code-block:: bash
+
+         $ sudo apt install datalad
+
+
+.. container:: toggle
+
+   .. container:: header
+
+      **2) Install within WSL2 [TODO]**
+
+   .. todo::
+
+      - find out how to install/enable WSL2
+
+      - find out what changes about the above instructions
 
 
 HPC environments or any system with singularity installed
```

Comment:
Remove this completely? The present target audience does not intersect much with HPC users. And those who want to can find this on the webpage.